lib/facility_locations.rb

```diff
@@ -0,0 +1,105 @@
+class FacilityLocations
+  attr_reader :facilities
+
+  def initialize
+    @facilities = []
+  end
+
+  def create_facilities(dmv_data)
+      dmv_data.each do |facility| 
+      hours(dmv_data)
+      phone(dmv_data)
+      address(dmv_data)
+      name(dmv_data)
+      holidays_closed(dmv_data)
+      facility = Facility.new(facility)
```

Comment:
I like the idea you're on here, with making all these helper methods to do specific things. The only problem I see is that basically, you're passing around this giant array of hashes to each of these methods, iterating through it, changing it and manipulating it, and if that array had a million or more elements in it, that would be extremely costly and slow. Instead, we can take pieces out of the array and then when we make changes, we can store that updated piece in a variable. 
For example, line 10 here, maybe it's something like `facility_hours = hours(dmv_data)` and then instead of returning that whole entire hash again, we just return the section of data that relates to the facility hours. Then we can sent those variables into Facility.new in a hash. 
Hoping that makes sense!